quarkus/tests/junit5/src/main/java/org/keycloak/it/utils/RawKeycloakDistribution.java

```diff
@@ -708,4 +714,39 @@ public <D extends KeycloakDistribution> D unwrap(Class<D> type) {
 
         throw new IllegalArgumentException("Not a " + type + " type");
     }
+
+    private static final class DefaultOutputConsumer implements OutputConsumer {
+
+        private final List<String> stdOut = Collections.synchronizedList(new ArrayList<>());
+        private final List<String> errOut = Collections.synchronizedList(new ArrayList<>());
+
+        @Override
+        public void onStdOut(String line) {
+            System.err.println(line);
```

Comment:
@stianst Shouldn't it be System.out.println(line);?

Reply:
It sure should. Fixing